packages/main/src/DateTimePicker.ts

```diff
@@ -384,6 +392,10 @@ class DateTimePicker extends DatePicker {
 		this._showTimeView = target.getAttribute("key") === "Time";
 		if (this._showTimeView) {
 			this._currentTimeSlider = "hours";
+			if (this._timeSelection!._currentSliderDOM) {
+				this._timeSelection!._currentSliderDOM.expanded = false;
```

Comment:
Accessing the internals of the TimeSelection looks wrong, we should think for more convenient APIs or for alternative solution
Moreover, setting expanded to false, then getting it back to true seems strange as well - to cause invalidation

Reply:
As we talked, I've changed the approach. I also tested in few scenarios and it seems to be working.